charts/lnd/values.yaml

```diff
@@ -103,3 +103,14 @@ rbac:
   create: true
 autoGenerateSeed:
   enabled: false
+
+postgresql:
+  # Enable LND on Postgresql, postgresql.uri becomes mandatory
+  enabled: true
+  # Deploy Postgresql with this chart inside Kubernetes
+  deploy: true
+  uri: postgres://test:test@lnd-postgresql:5432/lnd
```

Comment:
`postgresql.uri` needs to be given as a helm values input because it has to be written in configmap, is this a good practice?